src/components/Footer/Footer.module.scss

```diff
@@ -0,0 +1,122 @@
+@import "../../styles/utils/mixins";
+@import "../../styles/utils/vars";
+@import "../../styles/fonts/fonts.scss";
+
+* {
+  box-sizing: border-box;
+}
+
+.container {
+  position: fixed;
+  bottom: 0;
+  left: 0;
+  width: 100%;
+}
+
+.footer {
+  display: flex;
+  flex-direction: column;
+  justify-content: space-between;
+
+  height: 320px;
+  padding: 35px 16px;
+  box-shadow: 0 -1px 0 #e2e6e9;
+
+  @include onTablet {
+    justify-content: space-between;
+    flex-direction: row;
+    align-items: center;
+
+    padding: 32px;
+    height: 96px;
+  }
+
+  @include onDesktop {
+    padding: 32px 152px;
+  }
+
+  &__top-action {
+    display: flex;
+    align-items: center;
+    justify-content: center;
+
+    &-message {
+        font-weight: 300;
+        font-size: 12px;
+        line-height: 15px;
+        font-family: "Mont", sans-serif;
```

Comment:
use already-made typography for font-family etc and override special features (as 300 font-weight)